### PR TITLE
Upgrade play from 2.6 to 2.8

### PR DESCRIPTION
**Pull request by johnduffell**
This PR updates play from 2.6 to 2.8.  This means we will be on a supported version again.

### CODE REVIEW

File: conf/routes [hunk 1]
```diff
@@ -6,15 +6,15 @@ GET         /robots.txt                      controllers.CachedAssets.at(path="/
 GET         /favicon.ico                     controllers.CachedAssets.busted(path="images/favicons/32x32.ico")
 
 # Editions homepage
-GET         /                                controllers.Homepage.index
+GET         /                                controllers.Homepage.index()
```
Comment (johnduffell) [inline conf/routes:9]: I have added the brackets as per https://github.com/guardian/support-frontend/pull/3122#discussion_r653618017
allthough I can't remember the exact reason I went for adding the brackets here in place of removing them at the point of use.

File: app/MyApplicationLoader.scala [hunk 2]
```diff
@@ -31,7 +31,7 @@ class MyApplicationLoader extends ApplicationLoader {
       new MyComponents(context).application
     } catch {
       case e: Throwable =>
-        SafeLogger.error(scrub"Failed to load application due to $e")
+        SafeLogger.error(scrub"Failed to load application due to", e)
```
Comment (johnduffell) [inline app/MyApplicationLoader.scala:34]: this didn't provide a stacktrace before, so when it failed , I couldn't tell it was failing while loading the play config.

File: app/configuration/Config.scala [hunk 3]
```diff
@@ -17,8 +17,6 @@ object Config {
   val appName = "subscriptions-frontend"
   val config = ConfigFactory.load()
 
-  val playSecret = config.getString("play.crypto.secret")
```
Comment (johnduffell) [inline app/configuration/Config.scala:20]: deleted because it was unused and causing an exception on startup

File: conf/application.conf [hunk 4]
```diff
@@ -1,15 +1,6 @@
...
 # ~~~~~
 
-# Secret key
-# ~~~~~
-# The secret key is used to secure cryptographics functions.
-#
-# This must be changed for production, but we recommend not changing it in this file.
-#
-# See http://www.playframework.com/documentation/latest/ApplicationSecret for more details.
-play.crypto.secret="Is]F;gOuRS]X[eTv4j9k;_jC3Tc>2Xv<D_On/CRr/MBoJAM5yBB_w2Fp]1c7xDpX"
```
Comment (johnduffell) [inline conf/application.conf:11]: there was no need to have this default value, in fact it makes it more likely to accidentally use this (public) value in PROD. 
So i have removed and added the correctly named one for play 2.8 in the relevant secret files.

File: project/build.properties [hunk 5]
```diff
@@ -1 +1 @@
-sbt.version=1.3.2
+sbt.version=1.5.5
```
Comment (johnduffell) [inline project/build.properties:1]: always good to keep sbt bumped regularly

File: build.sbt [hunk 6]
```diff
@@ -61,7 +61,6 @@ libraryDependencies ++= Seq(
   "com.fasterxml.jackson.core" % "jackson-databind" % "2.10.5.1"
 )
 libraryDependencies ++= Seq(
-    cache,
```
Comment (johnduffell) [inline build.sbt:64]: doesn't seem to exist any more, but removing it doesn't break anything obvious

**Review by tomrf1 — APPROVED** [review]

**Pull request merged**